taxi/views.py

```diff
@@ -87,3 +88,38 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreationForm
```

Comment:
Ensure that the `DriverCreationForm` used here extends `UserCreationForm` instead of `forms.ModelForm`. This is crucial for maintaining consistency with user creation standards in Django as per the task requirements.